Ogury/src/main/java/com/mopub/mobileads/OguryInitializer.java

```diff
@@ -0,0 +1,75 @@
+package com.mopub.mobileads;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.mopub.common.MoPub;
+import com.mopub.common.Preconditions;
+import com.mopub.common.logging.MoPubLog;
+import com.mopub.mobileads.ogury.BuildConfig;
+import com.ogury.sdk.Ogury;
+import com.ogury.sdk.OguryConfiguration;
+
+import java.util.Map;
+
+import static com.mopub.common.logging.MoPubLog.AdapterLogEvent.CUSTOM;
+import static com.mopub.mobileads.OguryAdapterConfiguration.ADAPTER_NAME;
+import static com.mopub.mobileads.OguryAdapterConfiguration.ADAPTER_VERSION;
+
+public class OguryInitializer {
+    // Monitoring constants
+    private static final String MONITORING_KEY_MODULE_VERSION = "mopub_ce_version";
+    private static final String MONITORING_KEY_MOPUB_VERSION = "mopub_mediation_version";
+
+    private static boolean sInitialized = false;
+
+    public static boolean startOgurySDK(@NonNull Context context, @Nullable String assetKey) {
+        Preconditions.checkNotNull(context);
+
+        if (!OguryConfigurationParser.isAssetKeyValid(assetKey)) {
+            MoPubLog.log(CUSTOM, ADAPTER_NAME, "Ogury's initialization not started as the asset key is missing or empty.");
+            return false;
+        }
+
+        if (sInitialized) {
```

Comment:
We should not check that. We should always call Ogury.start() in order to pass through the monitoring code. We need to know which adapter classes are used. With this check, we can only know the first adapter class that is used. 
There is no problem to call Ogury.start() each time because all modules handle the multiple start calling